### PR TITLE
fix: static analysis issues

### PR DESCRIPTION
**Pull request by DrzwiPercepcji**
1. Fixed pub.dev static analysis issues:
obraz

### CODE REVIEW

File: lib/plist_parser.dart [hunk 1]
```diff
@@ -238,12 +238,12 @@ class PlistParser {
   }
 
   Map _handleDict(XmlElement elem) {
-    var children = elem.children.where(_isElement).cast<XmlElement>();
+    final children = elem.children.where(_isElement).cast<XmlElement>();
 
-    var keys =
-        children.where((el) => el.name.local == 'key').map((el) => el.text);
+    final keys =
+        children.where((el) => el.name.local == 'key').map((el) => el.innerText);
```
Comment (dirablue) [inline lib/plist_parser.dart:244]:
```suggestion
    final keys = children
        .where((el) => el.name.local == 'key')
        .map((el) => el.innerText);
```

fixed format with dart formatter

**Review by dirablue — CHANGES_REQUESTED** [review]
@DrzwiPercepcji 
Hi, Thank you for updating your code!
It's great.

it seems format check is failed, I think only one point I commented. could you check?

File: lib/plist_parser.dart (hunk 1 above)
Comment (DrzwiPercepcji) [reply lib/plist_parser.dart]: @dirablue fixed.

**Comment by dirablue** [comment]
@DrzwiPercepcji 
thanks! 
I published new version v0.0.11 to pub.dev
https://pub.dev/packages/plist_parser

**Pull request merged**